concepticondata/conceptlists/Carling-2017-127g.tsv

```diff
@@ -0,0 +1,128 @@
+Carling-2017-127g-14	14	1511	ROOSTER	rooster	12014	Farm animals	
+Carling-2017-127g-15	15	1318	CHICKEN	chicken	12015	Farm animals	
+Carling-2017-127g-16	16	1360	DUCK	duck	12016	Farm animals	
+Carling-2017-127g-17	17	2009	DOG	dog	12017	Farm animals	
+Carling-2017-127g-18	18	1208	CAT	cat	12018	Farm animals	
+Carling-2017-127g-19	19	665	BEE	bee	12019	Farm animals	
+Carling-2017-127g-20	20		*YEARLING (HORSE)	yearling	12020	Farm animals	
+Carling-2017-127g-21	21	522	WOLF	wolf	12021	Wild animals	
+Carling-2017-127g-22	22	1820	BEAR	bear	12022	Wild animals	
+Carling-2017-127g-23	23	3154	WILD BOAR	wild pig	12023	Wild animals	
+Carling-2017-127g-24	24	1139	LEOPARD	leopard	12024	Wild animals	
+Carling-2017-127g-25	25	1386	LION	lion	12025	Wild animals	
+Carling-2017-127g-26	26	1312	FOX	fox	12026	Wild animals	
+Carling-2017-127g-27	27		*JACKAL	jackal	12027	Wild animals	
+Carling-2017-127g-28	28	3142	BISON	bison	12028	Wild animals	
+Carling-2017-127g-29	29	1190	HARE	hare	12029	Wild animals	
+Carling-2017-127g-30	30	1136	RABBIT	rabbit	12030	Wild animals	
+Carling-2017-127g-31	31	730	SNAKE	snake	12031	Wild animals	
+Carling-2017-127g-32	32	2543	RAVEN	raven	12032	Wild animals	
+Carling-2017-127g-33	33	1905	EAGLE	eagle	12033	Wild animals	
+Carling-2017-127g-34	34	1936	DEER	deer	12034	Wild animals	
+Carling-2017-127g-35	35	964	WOOL	wool	12035	Products	
+Carling-2017-127g-36	36	580	FUR	fur	12036	Products	
+Carling-2017-127g-37	37	942	HONEY	honey	12037	Products	
+Carling-2017-127g-38	38	635	MILK	milk	12038	Products	
+Carling-2017-127g-39	39	1274	SALT	salt	12039	Products	
+Carling-2017-127g-40	40	1524	WINE	wine	12040	Products	
+Carling-2017-127g-41	41	941	MEAD	mead	12041	Products	(i.e. based on honey)
+Carling-2017-127g-42	42		*HOPS	hops	12042	Products	(Swe. humle)
+Carling-2017-127g-43	43	1931	BEESWAX	beeswax	12043	Products	
+Carling-2017-127g-44	44	634	MEAT	meat	12044	Products	
+Carling-2017-127g-45	45	323	FAT (ORGANIC SUBSTANCE)	grease	12045	Products	
+Carling-2017-127g-46	46	133	WEAVE	to weave	12046	Activities	
+Carling-2017-127g-47	47	121	SPIN	to spin (thread)	12047	Activities	
+Carling-2017-127g-48	48	1457	SEW	to sew	12048	Activities	
+Carling-2017-127g-49	49	1921	PLOUGH	to plough	12049	Activities	
+Carling-2017-127g-50	50	748	SOW SEEDS	to sow	12050	Activities	
+Carling-2017-127g-51	51	1077	WHEAT	wheat	12051	Crops & fruit	
+Carling-2017-127g-52	52	932	BARLEY	barley	12052	Crops & fruit	
+Carling-2017-127g-53	53	23	RYE	rye	12053	Crops & fruit	
+Carling-2017-127g-54	54	60	OAT	oats	12054	Crops & fruit	
+Carling-2017-127g-55	55		*LENTILS	lentils	12055	Crops & fruit	
+Carling-2017-127g-56	56	605	GRAIN	grain (generic)	12056	Crops & fruit	
+Carling-2017-127g-57	57	3062	TURNIP	turnip	12057	Crops & fruit	
+Carling-2017-127g-58	58	1320	APPLE	apple	12058	Crops & fruit	
+Carling-2017-127g-59	59	904	BERRY	berry	12059	Crops & fruit	
+Carling-2017-127g-60	60	920	GRAPE	grape	12060	Crops & fruit	
+Carling-2017-127g-61	61		*FLAX	flax	12061	Crops & fruit	
```

Comment:
Okay, sorry, more work now, but **all** concepts with an asterisk need to be added NOW to concepticon, otherwise, it doesn't make sense to provide them. If you add concepts with OR, you also need to make the linking. So this will take a bit more time, but you have the advantage of being able to make this quickly with grepping the new definitions from the concept lists. Also a consistency check, and I don't mind if you leave certain new conceps out, but then remove the starred gloss.

Reply:
Awesome, thanks for the feedback! Just wanted to 'do' the PR now so that I could work on this progressively. Thanks for reporting all the issues so quickly. :)

Reply:
@LinguList @Schweikhard see https://github.com/clld/concepticon-data/pull/564/commits/ecdbb943f68290c1948bd4453974da8a4303bf0d.